src/js/index.js

```diff
@@ -21,33 +21,19 @@ $(document).ready(() => {
 	}
 });
 
+const jqueryFieldSelectors = ['#address1', '#city', '#state', '#zipcode'];
+
 const validateAddress = () => {
 	var hasError = false;
-	if (!$('#address1').val()) {
-		$('#address1').parent().addClass('has-error');
-		hasError = true;
-	} else {
-		$('#address1').parent().removeClass('has-error');
-	}
 
-	if (!$('#city').val()) {
-		$('#city').parent().addClass('has-error');
-		hasError = true;
-	} else {
-		$('#city').parent().removeClass('has-error');
-	}
-
-	if (!$('#state').val()) {
-		$('#state').parent().addClass('has-error');
-		hasError = true;
-	} else {
-		$('#state').parent().removeClass('has-error');
-	}
-
-	if (!$('#zipcode').val()) {
-		$('#zipcode').parent().addClass('has-error');
-	} else {
-		$('#zipcode').parent().removeClass('has-error');
+	for (let i = 0; i < jqueryFieldSelectors.length; i++) {
```

Comment:
question: why do a for loop here but a foreach in `resetAddressFields`? Should be consistent, imo (ie: for of till we drop or use array methods, but not both).

I will approve this PR but something to keep in mind